src/workerd/api/r2-api.capnp

```diff
@@ -27,6 +27,8 @@ struct R2BindingRequest {
     uploadPart @10 :R2UploadPartRequest $Json.flatten();
     completeMultipartUpload @11 :R2CompleteMultipartUploadRequest $Json.flatten();
     abortMultipartUpload @12 :R2AbortMultipartUploadRequest $Json.flatten();
+    copy @13 :R2CopyRequest $Json.flatten();
```

Comment:
Note that the order of the declarations in the file does not have to match the numeric order. If you prefer to insert `copy @13` between `put @3` and `list@4`, you can.